llvm/include/llvm/TargetParser/AArch64CPUFeatures.inc

```diff
@@ -59,7 +59,7 @@ enum CPUFeatures {
   FEAT_SVE_F32MM,
   FEAT_SVE_F64MM,
   FEAT_SVE2,
-  FEAT_SVE_AES,
+  RESERVED_FEAT_SVE_AES, // previously used and now ABI legacy
```

Comment:
It would be nice to have a more complete explanation of these reserved values somewhere in these files.